be/src/util/system_metrics.cpp

```diff
@@ -435,4 +435,65 @@ void SystemMetrics::_update_fd_metrics() {
     fclose(fp);
 }
 
+int64_t SystemMetrics::get_max_io_util(
+        const std::map<std::string, int64_t>& lst_value, int64_t interval_sec) {
+    int64_t max = 0;
+    for (auto& it : _disk_metrics) {
+        int64_t cur = it.second->io_time_ms.value();
+        const auto& find = lst_value.find(it.first);
```

Comment:
find return a temporary variable, do not use reference

Reply:
ok